system-addon/test/unit/common/Actions.test.js

```diff
@@ -1,10 +1,26 @@
 const {
+  actionTypes: at,
   actionCreators: ac,
   actionUtils: au,
   MAIN_MESSAGE_TYPE,
-  CONTENT_MESSAGE_TYPE
+  CONTENT_MESSAGE_TYPE,
+  UI_CODE,
+  BACKGROUND_PROCESS,
```

Comment:
Ah, I had assumed that because they were exported, it was fair game for external callers to use them too.   Which is one reason it may be worth doing the _ stuff.  *shrug*